### ISSUE TITLE
Fake?

### ISSUE THREAD

**Issue by ProminenceClient** [quivings/Solara#37]
![image](https://github.com/quivings/Solara/assets/138956536/d687724f-70f3-4c04-97fc-7903b296f4f0)

**Comment by quivings** [comment]
Obviously

**Comment by xNasuni** [comment]
yeah that is not the real quivings, their real account name is just "quivinq" on discord
i deobfuscated the script and it just uses this rce that will be patched whenever qui pushes his latest changes but it just downloads a rat from a discord attachment then runs it

##### it's just blankgrabber nothing special
